Analytics/Integrations/Kahuna/SEGKahunaIntegration.m

```diff
@@ -396,6 +396,7 @@ - (void)swizzleAppDelegateMethods
 - (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error
 {
     @try {
+        [[SEGKahunaPushMonitor sharedInstance] failedToRegisterPush:error];
         if ([SEGKahunaPushMonitor sharedInstance].kahunaInitialized) {
             [KahunaAnalytics handleNotificationRegistrationFailure:error];
         }
```

Comment:
This "if ([SEGKahunaPushMonitor sharedInstance].kahunaInitialized) {" should not be present. The "failedToRegisterPush" takes care of the IF condition internally.